architecture/additional_concepts/networking.adoc

```diff
@@ -94,32 +97,166 @@ round-robin between each pod.
 
 [[network-plugins]]
 
-== Network Plugins
+== Network Plug-ins
 
-{product-title} supports the same plugin model as Kubernetes for networking pods.
+{product-title} supports the same plug-in model as Kubernetes for networking pods. 
 ifdef::openshift-enterprise,openshift-origin[]
-The following network plugins are currently supported by {product-title}.
+The following network plug-ins are currently supported by {product-title}.
 endif::[]
 
 [[openshift-sdn]]
 
 === {product-title} SDN
 
-{product-title} deploys a software-defined networking (SDN) approach for connecting
-pods in an {product-title} cluster. The {product-title} SDN connects all
-pods across all node hosts, providing a unified cluster network.
+{product-title} deploys a software-defined networking (SDN) approach for
+connecting pods in an {product-title} cluster. The {product-title} SDN connects
+all pods across all node hosts, providing a unified cluster network.
 
 {product-title} SDN is automatically installed and configured as part of the
-Ansible-based installation procedure.  Further administration should not
-be required; however, xref:sdn.adoc#architecture-additional-concepts-sdn[further details on the design and
-operation of {product-title} SDN] are provided for those who are curious or
-need to troubleshoot problems.
+Ansible-based installation procedure. See the
+xref:sdn.adoc#architecture-additional-concepts-sdn[{product-title} SDN] section
+for more information.
 
 ifdef::openshift-enterprise,openshift-origin[]
 [[nuage-sdn]]
 === Nuage SDN for {product-title}
 
-Nuage Networks' SDN solution based on it's link:http://www.nuagenetworks.net/products/virtualized-services-platform/[Virtualized Services Platform] delivers
-highly scalable policy based overlay networking for pods in an {product-title} cluster. Nuage SDN can be installed and configured as a part of the
-Ansible-based installation procedure. More information about the xref:nuagesdn.adoc#architecture-additional-concepts-nuagesdn[design and integration] is provided for a deep-dive. The xref:../../install_config/configuring_nuagesdn.adoc#install-config-configuring-nuage-sdn[installation section] will have detailed information about how to install and deploy {product-title} with Nuage SDN.
+link:http://www.nuagenetworks.net/products/virtualized-services-platform[Nuage
+Networks'] SDN solution delivers highly scalable, policy-based overlay
+networking for pods in an {product-title} cluster. Nuage SDN can be installed
```

Comment:
Maybe make Nuage SDN a link to xref:../../architecture/additional_concepts/nuagesdn.adoc#architecture-additional-concepts-nuagesdn here, then remove the "See the Nuage SDN section for more information" blurb. Then, you can continue with "See the Advance Installation section...." blurb.